### ISSUE TITLE
Statische kaarten zoeken in archief intern geoportaal

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/geoportaal-test#692]
Net als bij geodata (issue #691) nu ook zoeken in het interne geoportaal zoeken op statische kaarten in het archief.

**Comment by gerardnienhuis** [comment]
Nu staat er:


Zoeken naar GIS bestanden
Zoeken naar GIS bestanden in archief
Zoeken naar Services
Zoeken naar Statische kaarten


Dat moet worden:

Zoeken naar GIS bestanden
Zoeken naar GIS bestanden in archief
Zoeken naar Services
Zoeken naar Statische kaarten
Zoeken naar Statische kaarten in archief

**Comment by SandroNeumann** [comment]
Op dit moment worden alleen de statische kaarten met status "gepubliceerd" geharvest. Daar moet dus "gearchiveerd" bijkomen, maar deze moeten dan vervolgens alleen intern getoond worden?

Probleem is alleen dat de status niet in de metadata staat, dus in de harvester weten we niet of het bestand gearchiveerd is of niet. Dus hoe gaan we dit kenbaar maken in de metadata? Met welke waarde in welk element?

**Comment by gerardnienhuis** [comment]
correct, alleen mogelijkheid tot zoeken in archief alleen tonen bij interne geoportaal (net zoals nu bij geodatasets).

De harvester weet de status niet, maar toch is die ergens in het systeem bekend. Wordt namelijk gebruikt bij de beheermodule van statische kaarten. Deze status moet gebruikt worden.

**Comment by SandroNeumann** [comment]
Dat is mijn punt. Die staat alleen in de beheermodule. Niet in de harvester want daar wordt de metadata uitgelezen en daar staat deze niet in. De enige manier is om de status in de metadata weg te schrijven, maar daar is geen element voor.

**Comment by gerardnienhuis** [comment]
vraag is dus hoe de informatie uit de beheermodule ook beschikbaar kan komen voor de harvester. Is er niet een element te gebruiken daarvoor. Heeft dublin core niet een element wat lijkt op "status".

**Comment by SandroNeumann** [comment]
Helaas niet.

Echter, bedenk ik me net dat je in de WebDAV programmatuur wel properties mee kunt sturen. We kunnen op die manier de status meesturen.

**Comment by gerardnienhuis** [comment]
Klinkt prima. Is hiermee de vraag dan uitvoerbaar?

**Comment by SandroNeumann** [comment]
Ja, op deze wijze moet het mogelijk zijn.
